recipes/bw2parameters/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "bw2parameters" %}
+{% set version = "0.7" %}
+{% set sha256 = "f9d6cd45488762a755ede5c11754d5acf790f1e8ab592a86babd02f91c08a87a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+
+  run:
+    - asteval
+    - astunparse
+    - numpy
+    - python >=3.6
+    - stats_arrays
+
+test:
+  imports:
+    - {{ name }}
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/brightway-lca/brightway2-parameters
+  license: BSD-2-Clause
+  license_file: LICENSE
+  summary: 'Library for storing, validating, and calculating with parameters. Designed to work with the Brightway2 life cycle assessment framework, but is generic enough to work in other use cases.'
```

The summary should be short:
```suggestion
  summary: Library for storing, validating, and calculating with parameters
  description: |
      Designed to work with the Brightway2 life cycle assessment framework, but is
      generic enough to work in other use cases.
```